R/AllGenerics.R

```diff
@@ -230,17 +230,18 @@ setReplaceMethod("nonSplicedReads", "FraserDataSet", function(object, value){
 #' @param x A \code{FraserDataSet} object
 #' @param i A integer vector to subset the rows/ranges
 #' @param j A integer vector to subset the columns/samples
-#' @param by a character (j or ss) definig if we subset by
+#' @param by a character (j or ss) defining if we subset by
 #'             junctions or splice sites
+#' @param drop Currently not used 
 #' @return A subsetted \code{FraserDataSet} object
 #' @examples
 #'     fds <- createTestFraserDataSet()
-#'     fds[1:10,2:3]
-#'     fds[,samples(fds) %in% c("sample1", "sample2")]
-#'     fds[1:10,by="ss"]
+#'     fds[1:10,2:3,,drop=FALSE]
```

Comment:
I see this as a problem. As we do not want to ask the user to always set the drop argument. So we have to make sure it works without that for the end user. Within the package I'm fine with this. But outside on the user facing side we should not rely on this. to work properly.